app/code/Magento/QuoteGraphQl/Model/Resolver/BillingAddress.php

```diff
@@ -44,7 +44,13 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
         $cart = $value['model'];
 
         $billingAddress = $cart->getBillingAddress();
-        if (null === $billingAddress) {
+        if (null === $billingAddress ||
```

Comment:
Could you please use `validateAddressFromSchema` method? I mean move it to the separate class from  `\Magento\QuoteGraphQl\Model\Resolver\ShippingAddresses` and reuse for both - shipping and billing.